Yoshi.podspec

```diff
@@ -8,7 +8,7 @@
 
 Pod::Spec.new do |s|
   s.name             = "Yoshi"
-  s.version          = "2.0.1"
+  s.version          = "2.0.2"
```

Comment:
Don't bother with that I bump it when I release depending what the release include